### ISSUE TITLE
All browsers - Colour contrast on job cards

### ISSUE THREAD

**Issue by gobyrne** [GCTC-NTGC/TalentCloud#566]
# Description
The blue we use on a black background (for example the Department name on a job card) does not meet accessibility requirements.

![image](https://user-images.githubusercontent.com/2494551/50610650-bb83a080-0ea1-11e9-9214-d454ac49958c.png)
